.github/dependabot.yml

```diff
@@ -47,6 +47,21 @@ updates:
         update-types:
           - "minor"
           - "patch"
+      typescript-eslint:
+        patterns:
+          - "@typescript-eslint/eslint-plugin"
+          - "@typescript-eslint/parser"
+          - "typescript-eslint"
+        update-types:
+          - "minor"
+          - "patch"
+      turbo:
+        patterns:
+          - "turbo"
+          - "eslint-plugin-turbo-repo"
```

Comment:
This is my fault but it is actually `eslint-plugin-turbo` :stuck_out_tongue:

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/pull/11356/commits/10fdcd700d725bae5ee541f9873c01f90a84c5af